### PR TITLE
Update Topdown_GlobalRs.Rmd

### PR DESCRIPTION
**Pull request by jinshijian**
Figure 1 and Figure 2 updated

### CODE REVIEW

**Comment by jinshijian** [comment]
The colors matched between Figure 1 and Figure 2if color by source 
__Figure 1__

image


__Figure 2__
image

File: Topdown_GlobalRs.Rmd [hunk 1]
```diff
@@ -686,7 +687,9 @@ rs_figure_data %>%
   coord_cartesian(xlim = c(50, 120)) +
   # reverse the color scale so that teal (color 1) is always observations, 
   # while pink is always implied fluxes
-  scale_fill_discrete("", h.start = 180) + scale_color_discrete(h.start = 180) +
+  # scale_fill_discrete("", h.start = 180) + scale_color_discrete(h.start = 180) +
+  scale_fill_discrete("", labels = c(expression(Canonical~Rs), expression("Rs"[GPP]~"          "))) +
```
Comment (bpbond) [inline Topdown_GlobalRs.Rmd]: `c(expression(Canonical~R[S]), expression(Rs[GPP]))` ?

Comment (bpbond) [reply Topdown_GlobalRs.Rmd]: What are the spaces? Are you trying to left-align the legend items? Use `+ theme(legend.text.align = 0)`.

File: Topdown_GlobalRs.Rmd [hunk 2]
```diff
@@ -402,7 +402,8 @@ gpp_figure_data %>%
   geom_density(stat = 'density', alpha = 0.5) +
   theme(legend.position = c(0.75, 0.35)) +
   coord_cartesian(xlim = c(70, 250)) +
-  scale_fill_discrete("") + ylab("Density") +
+  scale_fill_discrete("", labels = c(expression(Canonical~GPP),expression(GPP[Rs]~"            "))) +
```
Comment (bpbond) [inline Topdown_GlobalRs.Rmd]: `c("Canonical GPP", expression(GPP[Rs]))`

Also what are the spaces? Are you trying to left-align the legend items? Use `+ theme(legend.text.align = 0)`.

File: Topdown_GlobalRs.Rmd [hunk 3]
```diff
@@ -686,7 +687,9 @@ rs_figure_data %>%
   coord_cartesian(xlim = c(50, 120)) +
   # reverse the color scale so that teal (color 1) is always observations,
```
Comment (bpbond) [inline Topdown_GlobalRs.Rmd]: This comment now no longer applies?

File: Topdown_GlobalRs.Rmd [hunk 4]
```diff
@@ -863,23 +873,25 @@ myarrow <- arrow(length = unit(0.1, "inches"))
 # for ggplot color defaults - that's how I got color codes below
 
 p_joint <- ggplot(joint, aes(GPP, Rs, color = Source)) + geom_point(alpha = 0.05) + geom_density2d() +
-  scale_color_discrete("Bootstrap\ndata source", labels = c(expression(GPP), expression(R[S]))) +
+  scale_color_discrete("", labels = c(expression(Canonical~GPP~vs~Rs[GPP]), 
+                                                            expression(Canonical~R[S]~vs~GPP[Rs]))) +
```
Comment (bpbond) [inline Topdown_GlobalRs.Rmd]: I don't think "vs" is the right terminology to use in the legend. The colors represent whether the joint distribution is of data driven by GPP or Rs.

**Comment by jinshijian** [comment]
For figure 1, if I add the code to change the legend, then the color will convert, @bpbond  can you please take a look? 
Line 691: 
  # scale_fill_discrete("", labels = c(expression(Canonical~Rs), expression("Rs"[GPP]~"          "))) +

**Comment by bpbond** [comment]
@jinshijian :
```R
  scale_fill_manual("", 
                    labels = c(expression(Canonical~Rs), expression(Rs[GPP])),
                    values = c("#00BFC4", "#F8766D")) +
```

**Pull request merged**